src/Call-Apis/addUser.js

```diff
@@ -0,0 +1,26 @@
+import axios from "axios";
+export const addUser = async (
+  tempEmail,
+  tempPassword,
+  dispatchAuth,
+  navigate
+) => {
+  try {
+    const res = await axios({
+      method: "POST",
+      url: "/api/auth/login",
+      data: { email: tempEmail, password: tempPassword },
+    });
+    if (res.status === 200) {
+      dispatchAuth({
+        type: "GET_USER_DETAILS",
+        payload: res.data.createdUser,
+      });
+      localStorage.setItem("token", res.data.encodedToken);
+      navigate("/");
+    }
+  } catch (e) {
+    console.log("error occured:  ", tempEmail, tempPassword, e);
```

Comment:
Remove the console.log if not required.

Reply:
The console is required so as to check for errors 
I always use console in catch